platform/linux-generic/odp_pool.c

```diff
@@ -346,7 +346,10 @@ static odp_pool_t pool_create(const char *name, odp_pool_param_t *params,
 		break;
 
 	case ODP_POOL_PACKET:
-		headroom    = CONFIG_PACKET_HEADROOM;
+		if (params->pkt.headroom < CONFIG_PACKET_HEADROOM)
+			headroom = params->pkt.headroom;
+		else
+			headroom    = CONFIG_PACKET_HEADROOM;
```

Comment:
Since `CONFIG_PACKET_HEADROOM` is what's currently returned as `min_headroom`, this logic is incorrect.

Reply:
Yes. This logic was based on previous assumption to make headroom parameter as a defined value rather than min. Will change it accordingly.